frontend/lib/src/components/elements/IFrame/IFrame.tsx

```diff
@@ -60,7 +60,8 @@ export default function IFrame({
 
   return (
     <iframe
-      data-testid="stIFrame"
+      className="stIframe"
```

Comment:
I am wondering whether some custom components rely on the existing `className` for styling purposes. Maybe we want to leave this one untouched for now to be on the safe-side. What do you think?

Reply:
I changed it back since the older name also is fine for our guidelines.

Reply:
But I think its the best timing to break names if something doesn't follow our naming schema